Makefile

```diff
@@ -1,2 +1,2 @@
 test-integration: ## Simple target running a kubectl command to ensure the cluster is up and running
-	kubectl get pods --all-namespaces -o wide
+	kubectl --kubeconfig=$(kind get kubeconfig-path) get nodes && kubectl --kubeconfig=$(kind get kubeconfig-path) get pods --all-namespaces -o wide
```

Comment:
Would you mind adding a comment mentioning why we don't use the `KUBECONFIG` environment variable? Something like `Environment variables are not always recognized by Makefiles, so it's recommended to use the --kubeconfig flag instead` could be okay.

Reply:
Sure, done!